src/test/kotlin/no/nav/familie/tilbake/config/KafkaErrorHandlerTest.kt

```diff
@@ -1,57 +0,0 @@
-package no.nav.familie.tilbake.config
-
-import io.kotest.assertions.throwables.shouldThrow
-import io.kotest.matchers.string.shouldContain
-import io.kotest.matchers.string.shouldNotContain
-import io.mockk.MockKAnnotations
-import io.mockk.impl.annotations.InjectMockKs
-import io.mockk.impl.annotations.MockK
-import org.apache.kafka.clients.consumer.Consumer
-import org.apache.kafka.clients.consumer.ConsumerRecord
-import org.junit.jupiter.api.BeforeEach
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.TestInstance
-import org.springframework.kafka.listener.MessageListenerContainer
-
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
-class KafkaErrorHandlerTest {
```

Comment:
Går det greit å slette denne? Har gått over til å bruke `familie-felles` sin `KafkaErrorHandler` i stedet.

Reply:
Kan testen leve videre, men med KafkaErrorHandler fra familie-felles? Eller funker det dårlig?

Reply:
Fikk problemer som jeg ikke skjønte så mye av. Fikk feil noe ala `couldn't find constructor for Task` ved oppstart. Mulig `InjectMockKs` annotasjonen ikke funker så bra med kode som er importert inn fra en avhengighet. Kunne kanskje gjort et forsøk på å gjøre det om til en `OppslagSpringRunnerTest()`, men var litt usikker på om det var verdt tiden for å teste kode som er felles.